packages/client/src/components/GameSettings/GameSettings.tsx

```diff
@@ -0,0 +1,42 @@
+import { StFlex } from 'styles/global';
+import GameSettingsButton from './GameSettingsButton';
+
+interface GameSettingsType {
+  handleCloseModal: () => void;
+}
+
+const settingsInputsValue = [
+  {
+    value: 2,
+  },
+  {
+    value: 4,
+  },
+];
```

Comment:
Если нет больше значений, может проще массив примитивов сделать?
```suggestion
const settingsInputsValue = [2, 4];
```